Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -324,6 +324,14 @@ def check_cpuid_instruction(formula)
       cpuid_instruction?(file, objdump)
     end
 
+    hardlinks = Set.new
+    return if formula.lib.directory? && formula.lib.find.any? do |pn|
+      next false if pn.symlink? || pn.directory? || pn.extname != ".a"
```

Comment:
I think we can use `ruby-macho` to identify static libraries. (Or otherwise parse the header manually...)